thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/utils/AnomalyUtils.java

```diff
@@ -66,4 +66,22 @@ public static Boolean checkHasLabels(List<MergedAnomalyResultDTO> mergedAnomalyR
     }
     return false;
   }
+
```

Comment:
Where and how this metaDataNode will be used? Does it have broader use cases? You may give it a more intuitive name. Also, even thought this is sub class, its better to name it 'MetaDataNode' to reflect that this is a class name, not a variable name.